ci-operator/config/ViaQ/logging-fluentd/ViaQ-logging-fluentd-release-5.6.yaml

```diff
@@ -32,7 +32,7 @@ base_images:
     namespace: logging
     tag: logging-elasticsearch6
   logging-eventrouter:
-    name: 5.y
+    name: 6.y
```

Comment:
Normally I would suggest making this a specific 5.x image but given the ER doesn't really change and 5.6 is going away in a few months this is fine.